src/hooks/docs.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Code, Icon, Text } from '@chakra-ui/react';
+
+import { useRtl } from '@/hooks/useRtl';
+
+export default {
+  title: 'Hooks/useRtl',
+};
+
+export const Default = () => {
+  const { direction, rtlValue } = useRtl();
+
+  return (
+    <Box>
+      <Text>
+        This hook is returning the direction of the text and the rtlValue (rtl
+        meaning Right to Left) which can have either value 'rtl' or value 'ltr'.
```

Comment:
What is the difference between the direction of the text and the rtlValue ?

To prevent such confusion, you may explain how those values are used instead of what they mean.